modules/tests/support/model_based/README.md

```diff
@@ -0,0 +1,38 @@
+## Model-based tests for IBC modules
```

Comment:
I would move this readme to a higher-level, inside `modules/tests/`. Here it's a bit buried and it's easy to overlook it.

Reply:
Done in 5e9dd634a3bc04c0f193ab3872b83dc00505d1d4.